tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -10,6 +10,13 @@
       "op":"replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/1/env/0",
       "value": {"name": "RELATED_IMAGE_ANSIBLEEE_IMAGE_URL_DEFAULT", "value": "{{ ansibleee_runner_img | default('quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest')}}"}}]'
 
+- name: Wait for the ansible-operator to restart with the new ENV
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc wait -n openstack-operators --timeout=120s --for=delete $(oc get -n openstack-operators pod -l app.kubernetes.io/name=openstack-ansibleee-operator -o name)
```

Comment:
this should work short-term. Long term, CI should only wait for some ready status perhaps, and aeee operator should not yield it unless deployed the image. The same story as minor updates for service operators, we don't want eventual consistent playbooks deployed, but only a final version

Reply:
> this should work short-term. Long term, CI should only wait for some ready status perhaps, 

At the moment I don't see any ready status that I can track meaningfully. Deployment has observedGeneration and it changing early (as we expect), but none of the other status fields incl conditions changes while the new operator pod is rolled out. (or I'm missing something). The CSV has ` phase` status field that transition between Installing and Succeeded, but CSV has no observedGeneration field, so I cannot check if it is succeeded on the latest spec update.

> and aeee operator should not yield it unless deployed the image. 

It is not like that the image is deployed right away. The image is used to define jobs for the dataplane services created after the reconfiguration. So I don't see what yielding can be prevented here.